src/app/modules/angular-slickgrid/components/angular-slickgrid.component.ts

```diff
@@ -168,6 +168,20 @@ export class AngularSlickgridComponent implements AfterViewInit, OnDestroy, OnIn
   @Input() gridId: string;
   @Input() gridOptions: GridOption;
 
+  @Input()
+  get paginationOptions(): Pagination | undefined {
+    return this._paginationOptions;
+  }
+  set paginationOptions(options: Pagination | undefined) {
+    if (options && this._paginationOptions) {
+      this._paginationOptions = { ...this._paginationOptions, ...options };
+    } else {
+      this._paginationOptions = options;
+    }
+    this.gridOptions.pagination = options;
```

Comment:
this should be replaced with `this.gridOptions.pagination = this._paginationOptions;`